src/audio/rtnr/rtnr.c

```diff
@@ -510,64 +514,53 @@ static int rtnr_reconfigure(struct comp_dev *dev)
 	return 0;
 }
 
-static int rtnr_set_comp_config_by_ipc_config_process(struct comp_dev *dev,
-							struct ipc_config_process *config)
+static int rtnr_set_config_data(struct comp_dev *dev,
```

Comment:
suggest to rename as "rtnr_set_config_bytes" to be more clear that this function is for rtnr_bytes control

Reply:
Fixed, thanks.